src/governance-vote.cpp

```diff
@@ -239,7 +259,21 @@ bool CGovernanceVote::IsValid(bool useVotingKey) const
         return false;
     }
 
-    return CheckSignature(useVotingKey ? infoMn.keyIDVoting : infoMn.keyIDOperator);
+    if (useVotingKey) {
+        CheckSignature(infoMn.keyIDVoting);
+    } else {
+        if (deterministicMNManager->IsDeterministicMNsSporkActive()) {
```

Comment:
combine to one line

Reply:
In this case, I actually left in some unfinished garbage by mistake. Added a commit on-top.